index.js

```diff
@@ -44,8 +44,22 @@ export async function placement (req, res) {
     const bigquery = new BigQuery()
     const [job] = await bigquery.createQueryJob(options)
     console.log(`Job ${job.id} started.`)
-    const [rows] = await job.getQueryResults()
-    await bqHandler(rows)
+    async function manualPaginationCallback (err, rows, nextQuery, apiResponse) {
+      await bqHandler(rows)
+      if (nextQuery) {
+        // More results exist.
+        job.getQueryResults(nextQuery, manualPaginationCallback)
```

Comment:
I believe ```nextQuery``` will have the same options as to when we originally call ```getQueryResults```, and then it will be ```null``` if there are no more results.